docs/how_to/wizards.rst

```diff
@@ -49,6 +49,11 @@ do this for ``MyApp``, it might look like this::
 
     wizard_pool.register(my_app_wizard)
 
+.. note::
+
+    if your model doesn't define a ``get_absolute_url`` function then your wizard
```

Comment:
Capitalise *If*.

Reply:
Thanks @marksweb, I've left a number of comments, this seems to be one of those simple-looking fixes that uncovers a number of related issues. Would you care to tackle them?

Reply:
Yeah I'll have a look at it when I get a minute.

On a side note, I remember making that 'if' lowercase as it was consistent with the existing note I checked for reference, so I'll take a look over that to ensure consistency & sentence case.